SimMuon/MCTruth/python/NewMuonAssociatorByHits_cfi.py

```diff
@@ -0,0 +1,147 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
And in general- what is the impact on time/memory with pileup with the new selectors?